src/coreclr/tools/Common/TypeSystem/MetadataEmitter/TypeSystemMetadataEmitter.cs

```diff
@@ -472,7 +517,6 @@ void EncodeMethodSignature(BlobBuilder signatureBuilder, MethodSignature sig, Em
 
             signatureEncoder.MethodSignature(sigCallingConvention, genericParameterCount, isInstanceMethod);
             signatureBuilder.WriteCompressedInteger(sig.Length);
-            // TODO Process custom modifiers in some way
```

Comment:
Just for my education - does deletion of this comment mean that this has been fixed independently or that we no longer care about custom modifiers?

Reply:
We already wrote the code to handle custom modifiers a year or so ago, but never actually cleaned up this comment.

Reply:
In particular that's what the embedded signature data stuff was written for originally. I will say that it is completely and totally incomprehensible as Michal has noted before, but it does work. And with handling of array shapes we hopefully won't ever have to touch it again.